commcare_connect/templates/tailwind/pages/opportunity_dashboard/dashboard.html

```diff
@@ -0,0 +1,191 @@
+{% extends 'tailwind/base.html' %}
+{% load i18n %}
+
+{% block title %}{{ request.org }} - {{ object.name }}{% endblock %}
+
+{% block content %}
+    <div class="container md:max-w-screen-2xl mx-auto flex flex-col gap-4">
+      {% include 'tailwind/components/breadcrumbs.html' with path=path %}
+        <div class="flex flex-col gap-8 w-full rounded-lg p-4 {% if not object.is_test %}bg-brand-deep-purple{% else %}bg-white shadow-xs{% endif %}">
+          <div class="flex items-center {% if object.is_test %}gap-6{% endif %} relative" x-data="{ isOpen: false }">
+              <h3 class="text-sm font-medium text-brand-sky flex-1">{% if object.manged %}{{ object.program_name }}{% endif %}</h3>
+              {% if object.is_test %}
+              <span class="badge badge-md primary-light">Test</span>
+              {% endif %}
+              <i class="fa-light cursor-pointer {% if not object.is_test %}text-white{% else %}text-gray-800{% endif %}"
+                 :class="isOpen ? 'fa-xmark-large' : 'fa-bars'"
+                 @click="isOpen = !isOpen"></i>
+              <!-- Dropdown Menu -->
+              {% include "tailwind/pages/opportunity_dashboard/dropdown.html" with request=request opportunity=object program_manager=program_manager %}
+          </div>
+          <div class="flex justify-between items-end">
+              <div class="flex flex-col gap-2">
+                  <h1 class="text-2xl font-medium {% if not object.is_test %}text-white{% else %}text-brand-deep-purple{% endif %} block">{{ object.name }}</h1>
+                <p class="text-sm font-regular {% if not object.is_test %}text-gray-100{% else %}text-gray-600{% endif %} w-6/12">{{ object.description }}</p>
+              </div>
+              <div class="flex mt-6">
+                  <div class="flex gap-4" x-data="{
+                      showModal: false,
+                      selectedTab: '',
+                      selectedOption: '',
+                      toggleModal() {
+                          this.showModal = !this.showModal;
+                          document.body.style.overflow = this.showModal ? 'hidden' : 'auto';
```

Comment:
It's better to do it in a declarative way, and use tailwind instead of raw style

```
      x-init="$watch('showModal', value => {
          document.body.classList.toggle('overflow-hidden', value);
      })">
```